doc/user_guide/signal/generic_tools.rst

```diff
@@ -485,28 +481,31 @@ signal:
 
 .. code-block:: python
 
-    >>> s = hs.signals.Signal1D(np.ones((2, 5, 10), dtype=np.uint8)
+    >>> s = hs.signals.Signal1D(np.ones((2, 5, 10), dtype=np.uint8))
     >>> print(s)
     <Signal1D, title: , dimensions: (5, 2|10)>
     >>> print(s.data.dtype)
     uint8
 
-    # Use dtype=np.unit16 to specify a dtype
+    Use ``dtype=np.unit16`` to specify a dtype
```

Comment:
Here and in the two comments below, the formatting by "``" works only if these are not part of the code environment (i.e. separated code environments with comments as plain text would be needed)!

Reply:
code-blocks are not explicitly defined now, but the formatting does work correctly

Reply:
Yeah, I have seen this syntax used somewhere and I suspect that it may default to python. I haven't looked it up in the sphinx documentation, but if this is the case, this could be used more widely as this simplify a bit the syntax.

Reply:
Indeed, the code-block statements in the user guide and docstrings are redundant if `>>>` is used, and indents and blank lines are added correctly:

see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks
and https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#doctest-blocks